scripts/automation/s3-test/create_account.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+S3_USER=$1
+BUILD_NUMBER=$2
+
+add_common_separator() {
+    echo -e '\n--------------- '"$*"' ---------------\n'
+}
+
+check_status() {
+    return_code=$?
+    error_message=$1
+    if [ $return_code -ne 0 ]; then
+        add_common_separator ERROR: $error_message
+    exit 1
+    fi
+    add_common_separator SUCCESS
+}
+
+if [ "$2x" == "x" ]; then
+    echo "Please pass proper input"
+    exit 1
+fi
+
+if [ -f bearertoken.log ]; then
+    rm -rf bearertoken.log
+fi
+
+TOKEN=$(curl -s -i -k -d '{"username": "cortxadmin", "password": "Cortxadmin@123"}' https://s3.seagate.com:31169/api/v2/login|grep Authorization| grep Bearer|awk -F':' '{print $2}'|sed 's/ //'|sed -e "s/\r//g")
```

Comment:
Please remove the hardcoded username and password. Fetch it from solution.yaml.  Refer - https://github.com/Seagate/cortx-re/blob/main/solutions/kubernetes/io-sanity.sh#L49

Reply:
Can we take this part as a improvement?

Reply:
added credentials into Jenkins environment variables.